Source/Client/Windows/ServerBrowser.cs

```diff
@@ -711,6 +735,8 @@ public class SaveFile
 
         public int protocol;
 
+        public bool Valid => rwVersion != null;
```

Comment:
Validity of a save file is determined by whether or not it's version is not null. This is fine for now.